pkg/plugins/platform/local.go

```diff
@@ -157,6 +157,14 @@ func (l *LocalApp) ImportDB(imPath string, extPath string) error {
 	var extPathPrompt bool
 	dbPath := filepath.Join(l.AppRoot(), ".ddev", "data")
 
+	if preCmds := l.AppConfig.Commands["pre-import-db"]; len(preCmds) > 0 {
+		fmt.Println("Executing pre-import commands...")
+		err := l.ProcessHooks("pre-import-db")
+		if err != nil {
+			return err
+		}
+	}
```

Comment:
the above code stanza should be a function which accepts a command key and executes the commands. It's repeated ver-batum (besides the key value) a few times.